File: algo-expert/js/isValidSubsequence/isValidSubsequence.js
const isValidSubsequence = (array, sequence) => {
  let pointerIndex = 0; 
  for (let i=0; i<array.length; i++) {
    if (pointerIndex == sequence.length) {
      return true; 
    }
    if (array[i]==sequence[pointerIndex]) {
      pointerIndex++
    }
    console.log(pointerIndex)
  }
  return pointerIndex == sequence.length;
}

isValidSubsequence([5,1,22,25,6,-1,8,10], [25]); 

File: algo-expert/python/isValidSubsequence/isValidSubsequence.py
def is_valid_subsequence(array, sequence):
    array_index = 0
    sequence_index = 0
    while array_index < len(array) and sequence_index < len(sequence):
        if array[array_index] == sequence[sequence_index]:
            sequence_index += 1
        array_index += 1
    return sequence_index == len(sequence)
is_valid_subsequence([5,1,22,25,6,-1,8,10], [25,10]);